modules/accredited_representative_portal/app/serializers/accredited_representative_portal/power_of_attorney_request_resolution_serializer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequestResolutionSerializer
+    include JSONAPI::Serializer
+
+    attribute :id
+
+    attribute :type do |object|
+      object.resolving_type.demodulize.underscore.split('_').last
+    end
```

Comment:
Once we have the polymorphic variants, each of them can hardcode this rather than manipulate the name of the Ruby class. In the end we should get: `"type": "decision"` and `"type": "expiration"`.